srcStatic/op2extStatic.vcxproj

```diff
@@ -167,7 +167,7 @@
     <ClInclude Include="IniFile.h" />
     <ClInclude Include="LocalResource.h" />
     <ClInclude Include="Log.h" />
-    <ClInclude Include="Log/Logger.h" />
+    <ClInclude Include="Log\LogDestination.h" />
```

Comment:
What's up with the path separator on this line? It doesn't match the others.

Reply:
I believe Visual Studio will default to using the Windows specific path separator when automatically editing the filters file, although it will work with either path separator. I'd like to default to using the Windows specific "\" since the IDE seems to generate it when updating tracked files. I suspect all the "/" path separators were added by manual edits to the filters file instead of the automated edits, although I'm not really inclined to research at this time.

Reply:
I wonder if an earlier version of Visual Studio used a different path separator. I imagine people editing on Windows would have added files through Visual Studio, and people editing from Linux would just copy/paste from what was already present.

At any rate, it should probably be using the Visual Studio default.